src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+class KlarnaCreditPromotionBadge extends React.Component {
+  componentDidMount () {
+    const {
+      klarnaExpressCheckoutScriptSource,
+      klarnaMerchantId,
+      klarnaEnvironment
+    } = this.props
+  
+    // In order for the klarna-express-button markup to be hydrated properly,
+    // this script has to be loaded each time this component renders
+    const script = document.createElement('script')
+    script.src = klarnaExpressCheckoutScriptSource
+    script.id = 'klarnaCheckoutButtonScript'
+    script.setAttribute('data-id', klarnaMerchantId)
+    script.setAttribute('data-environment', klarnaEnvironment)
+    script.async = true
+    document.head.appendChild(script)
+  }
+
+  componentWillUnmount () {
+    document.querySelector('#klarnaCheckoutButtonScript').remove()
```

Comment:
Would it make sense to store a reference to the `script` tag so we don't have to add an `id` and query the document?

```js
componentDidMount () {
  ...
  this.script = document.createElement('script')
  ...
}

componentWillUnmount () {
  this.script.remove()
}
```